client/components/Student.react.js

```diff
@@ -15,7 +16,7 @@ class Student extends React.Component {
           </button>
         </td>
         <td>
-          <button className="ui red icon button">
+          <button className="ui red icon button" onClick={this.props.handleDelete.bind(this, studentId)}>
```

Comment:
let's use `onClick={() => this.props.handleDelete(studentId)}`, think its easier for readers to understand (don't need to know "bind")